src/ensembl/src/content/app/custom-download/containers/content/result-holder/resultHolderHelper.ts

```diff
@@ -1,81 +1,96 @@
 import config from 'config';
 
-export const getEndpointUrl = (attributes: any) => {
-  let endpoint = config.genesearchAPIEndpoint + '/genes/query?query=';
-
-  let endpointFields = '';
-  attributes.forEach((attribute: any) => {
-    endpointFields += attribute[2] + ',';
+import { keys, mapKeys, set, get, trim } from 'lodash';
```

Comment:
No. This imports the whole lodash library.

Reply:
Updated it